docs/source/reference/tpu.rst

```diff
@@ -131,63 +192,79 @@ With the above YAML, you should be able to launch the training job with :code:`s
 
 
 
-TPU VMs
+
+
+
+TPU Pods
 --------------------------------
 
-To use TPU VMs, user only needs to add :code:`tpu_vm: True` and the desired TPU runtime version in :code:`accelerator_args` shown below:
+A `TPU Pod <https://cloud.google.com/tpu/docs/training-on-tpu-pods>`_ is a collection of TPU devices connected by dedicated high-speed network interfaces for scalable training.
+User can simply change the accelerator name to a TPU Pod (e.g., :code:`v2-8` -> :code:`v2-32`) in YAML to launch such resources.
 
 .. code-block:: yaml
+   :emphasize-lines: 2-2
 
    resources:
-      accelerators: tpu-v2-8
+      accelerators: tpu-v2-32
       accelerator_args:
          runtime_version: tpu-vm-base
          tpu_vm: True
 
+.. note::
 
-Note that :code:`instance_type` is no longer needed because TPU VMs is a standalone host VM that physically connects to the TPU device.
+   Both TPU architectures, TPU VMs and TPU Nodes, are supported. Our example is based on TPU VM.
 
-Now we show an example of running `mnist training <https://cloud.google.com/tpu/docs/run-calculation-jax#running_jax_code_on_a_tpu_vm>`_ on TPU VM with JAX.
+To show all available TPU Pods, run :code:`sky show-gpus`:
+
+.. code-block:: console
+
+   GOOGLE_TPU   AVAILABLE_QUANTITIES
+   tpu-v2-8     1
+   tpu-v2-32    1
+   tpu-v2-128   1
+   tpu-v2-256   1
+   tpu-v2-512   1
+   ...
+
+After creating a TPU Pod, multiple host VMs (e.g., :code:`v2-32` comes with 4 host VMs) will be ready.
```

Comment:
This is somewhat confusing. Earlier we explained "host VMs" only appear for TPU Nodes. Is this word sed in the official TPU Pod docs, even for TPU VM+Pod? Should we change it to "worker IPs"?

Reply:
Yes it's confusing.. But host is the term used in GCP document. 
https://cloud.google.com/tpu/docs/jax-pods

Indeed both TPU Nodes and TPU VMs have the concept of "Host VM" but the former can be paired with any instance type you want and the latter's host instance type is fixed.
Does that clarify it?